mariner-installer/build/build

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -ex
+
+echo "BRANCH: $BRANCH"
+cd $WORKSPACE/mariner-installer
+
+. /etc/os-release
+
+DEBIAN=0
+
+if [ "$ID" == 'ubuntu' ] ; then
+    DEBIAN=1
+fi
+
+if [ $DEBIAN = 1 ] ; then
+    dpkg-buildpackage -us -uc
+else
+    tito build --rpm
+fi
+
+if [ -n "$BRANCH" ]; then
+    # A user explicitly told Jenkins to build a branch.
+    # (ie, this is not an automatic GitHub pull request build.)
+    # Move the artifacts into position for archiving.
+    cd $WORKSPACE
+    rm -rf $BRANCH
+    mkdir $BRANCH
+
+    if [ $DEBIAN = 1 ] ; then
```

Comment:
TL;DR - I don't think these packages are destined for ceph.com

I think this comes from the fact that we are only building this for Trusty?  That's the only dist in the job config.

I think this job config is mostly for a smoke test.  A separate job will come in for downstream packaging (for RHCS on Ubuntu).

For community packages, I intend to host Deb packages in a PPA, just like the RPMs are in a COPR repo.  

That's my story.  Hopefully @ktdreyer agrees.